pol-core/bscript/compiler/codegen/InstructionGenerator.cpp

```diff
@@ -243,6 +247,23 @@ void InstructionGenerator::visit_branch_selector( BranchSelector& node )
   }
 }
 
+void InstructionGenerator::visit_class_declaration( ClassDeclaration& node )
```

Comment:
This may be removed in scoping PR, as (1) variables will be added to top level statements and (2) user functions will be added through the normal "emit user functions" code generation process